xml/vt_best_practices.xml

```diff
@@ -2950,6 +2946,8 @@ virtio_rng.0</screen>
 
   <sect2 xml:id="vt.best.perf.mtype">
    <title>Updating the Guest Machine Type</title>
+   <!-- cwickert 2017-01-26: The sentences in the next two paragaphs are quite
+    long and hard to read/understand. Try to make them shorter? -->
```

Comment:
&qemu; machine types define details of the architecture that are particularly relevant during migration and session management. The list of machine types is constantly maintained&mdash;the addition of new features and fixes results in new machine types.
Old machine types remain in the list for compatibility reasons when new versions are released. However, to take advantage of improvements, it is recommended to always migrate to the latest machine type when upgrading.